src/app/use-cases/status-signature-request.ts

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable sonarjs/no-small-switch */
```

Comment:
you usually want to put this kind of comments near the code selectively disablling "next line" (otherwise when you refactor your code and  the switches are deleted (for example) these comments will remain even if unused.